source/_posts/2019-10-07-upgrading-psr-interfaces.md

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Upgrading PSR interface'
+description: >-
+Greetings, PHP. The PHP Framework Interoperability Group is **actively looking for feedback on a way to safely update and modernize several existing PSRs**. We think this is a workable approach, but before embarking on it we want to get feedback from the broader community. That means you.
+date: '2019-10-07T14:11:13.604Z'
+categories: []
+keywords: []
+slug: upgrading-psr-interfaces
+tags:
+ - php-fig
+ - bylaws
+author: crell
+layout: post
+use:
+ - authors
+ - posts
+---
+
+## PHP-FIG wants your help to modernize PSRs!
+
+Greetings, PHP. The PHP Framework Interoperability Group is **actively looking for feedback on a way to safely update and modernize several existing PSRs**. We think this is a workable approach, but before embarking on it we want to get feedback from the broader community. That means you.
```

Comment:
I gotta say, I really, really loathe the excessive bolding in this format.  It's annoying enough when comic books use it in place of emotion, but in a blog post it's just super tacky.  Please remove.

Reply:
I personally use that not as an aesthetic thing but as a tool to highlight stuff in long text. @iansltx @goetas WDYT?

Reply:
For me is fine with bold. It will look as highlighing some kind of "call to action"... that is exactly what the blog post is about.

Reply:
There is only one call to action in the entire post, and that's at the very end.  The bolding of every third sentence is just amateur looking.